src/components/Process/Aside.tsx

```diff
@@ -37,23 +37,22 @@ const ProcessAside = () => {
     !(election?.electionType.anonymous && voting)
   const showVotes = !election?.electionType.secretUntilTheEnd || election.status === ElectionStatus.RESULTS
 
-  let totalWeight = 0
+  let votes = 0
   if (election && showVotes) {
     const decimals = (election.meta as any)?.token?.decimals || 0
-    const totalsAbstain = election?.questions.map((q) => ('numAbstains' in q ? Number(q.numAbstains) : 0))
-
-    totalWeight = election?.questions
-      .map((el, idx) => el.choices.reduce((acc, curr) => acc + Number(curr.results), totalsAbstain[idx]))
-      .map((votes: number) => results(votes, decimals))
-      .reduce((acc, curr) => acc + curr, 0)
+    // It just has to check the first question to get the total of votes.
+    const question = election?.questions[0]
```

Comment:
Note all the previous operations could work if `questions` or `choices` were empty, because they were maps and reduces, but due to this change, and considering there's no `election?.questions.length` check, this will fail if the election has no questions defined.

Reply:
And this code is to count votes, so if there are no questions, where is supposed i can find votes?